pkg/liqoctl/offload/handler.go

```diff
@@ -139,6 +192,18 @@ func (o *Options) output() error {
 		},
 	}
 
+	// Print the object to a buffer to capture YAML output
+	if o.OutputFormat == "yaml" {
```

Comment:
If you move the definition of the `nsoff` variable just above the switch...case, I believe we can avoid this if condition, and place its content inside the "yaml" case of the switch...case above